### DISCUSSION TITLE
vscode-tweaker: Unable to get it working!?

### DISCUSSION THREAD

**Discussion by silopolis** [ramensoftware/windhawk-mods#230] [General]
Hi,

Like many others, I believe, I discovered your software thanks to MS not taking care of a recent issue that no one seems to care about https://github.com/microsoft/vscode/issues/519#issuecomment-1196038396 ! :|

Anyway, I'm glad I know Windhawk now, and thank you for making it Open Source 🙏 

That said, I'm unable to get it working with VSCodium :'(

I installed Windhawk v1.3.1 beta using Chocolatey, then installed the `notepad-dark-mode` and `vscode-tweaker` mods.
Notepad module works and appears in loaded modules when I launch it.
OTOH, I can't get CSS injected in VSCodium, and nothing appears in the loaded modules window when I launch it.

Could this be because the mod targets `code.exe` whereas mine in `VSCodium.exe`?

Thanks in advance for your support 🙏 

TY
J

**Comment by m417z** [comment]
Hi,

> Could this be because the mod targets `code.exe` whereas mine in `VSCodium.exe`?

Yes, currently the mod only targets `code.exe`, which means that VSCode is supported while VSCodium isn't. You can add `VSCodium.exe` manually in the advanced tab of the mod. Note that it will also affect the Windhawk UI which is based on VSCodium. Alternatively, you can use the full path of your `VSCodium.exe` file to keep Windhawk unaffected.
